assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -1313,6 +1313,28 @@ describe("DiversionPage", () => {
     ).toBeVisible()
   })
 
+  test("'View Draft Detour' screen allows user to edit detour text", async () => {
+    const { container } = render(<DiversionPage />)
+
+    act(() => {
+      fireEvent.click(originalRouteShape.get(container))
+    })
+
+    act(() => {
+      fireEvent.click(originalRouteShape.get(container))
+    })
+
+    await userEvent.click(reviewDetourButton.get())
+
+    const input = screen.getByRole("textbox") as HTMLInputElement
+
+    act(() => {
+      fireEvent.change(input, { target: { value: "Hello World" } })
+    })
```

Comment:
Do you think this could be changed to use [`type()` from the user-event library](https://testing-library.com/docs/user-event/utility#type)?